lib/helpers/db.js

```diff
@@ -0,0 +1,24 @@
+'use strict'
+
+const { dbUsers } = require('config')
+
+const db = require('moonbeam-mongodb')(dbUsers)
+
+exports.start = () =>
+  new Promise((resolve, reject) => {
+    db.start(err => {
+      if (err) return reject(err)
+      exports.tradesCollection = db.db.collection(dbUsers.collectionTrades)
```

Comment:
maybe it's better to have something like this:

```javascript
const collections = {}
exports.getCollection = (collection) => collections[collection] || null

exports.start = () =>
  new Promise((resolve, reject) => {
    db.start(err => {
      if (err) return reject(err)
      collections['tradesCollection'] = db.db.collection(dbUsers.collectionTrades)
      resolve()
    })
  })
```

Reply:
sure, we can to that, although right now there is only 1 collection